src/app/shared/services/http-v2/http-v2.service.ts

```diff
@@ -61,13 +61,15 @@ export class HttpV2Service {
     data: any = {},
     responseClass?: ResponseClass<T>,
     options: RequestOptions = defaultOptions,
+    responseType?: 'json' | 'text',
```

Comment:
Thank you for adding this new feature to the HttpV2Service. I think ideally I'd like to see `responseType` in the `options` object instead of a positional argument. It makes our public methods have up to five arguments and some internal methods have even more. This can be annoying when we want to make a basic request with default options but we need to adjust the response type because we have to fill out every argument that would have been optional. (I regret having options come *after* `resposneClass` since those options are typically set more than the `responseClass` is).